controllers/machinehealthcheck_controller_test.go

```diff
@@ -342,7 +342,8 @@ var _ = Describe("MachineHealthCheck Reconciler", func() {
 				for _, m := range rtc.expectRemediated() {
 					machine := &clusterv1.Machine{}
 					key := types.NamespacedName{Namespace: m.Namespace, Name: m.Name}
-					Expect(k8sClient.Get(ctx, key, machine)).ToNot(Succeed())
+					Expect(k8sClient.Get(ctx, key, machine)).To(Succeed())
+					Expect(machine.GetAnnotations()).To(HaveKeyWithValue(clusterv1.MachineUnhealthy, ""))
```

Comment:
```suggestion
					Expect(machine.GetAnnotations()).To(HaveKey(clusterv1.MachineUnhealthy))
```

Reply:
This was I change I had suggested (to check the value) since we are expecting it to be empty (based on the proposal), there had been talk of putting a value in this but that was decided against, I don't feel strongly about it though if you'd prefer not to check the value

Reply:
In the past we've removed explicit checks for empty string for keys where we only really care that the key is present. Unless we really care about the value for some reason I think we should stick with a key-based check.